File: src/main/scala/com/algorithmlx/inscribers/init/InscribersRecipeTypes.scala
package com.algorithmlx.inscribers.init

import com.algorithmlx.inscribers.recipe.{CastRecipeType, InscriberRecipe}
import net.minecraft.item.crafting.IRecipeType

object InscribersRecipeTypes {
  val inscriberRecipe = CastRecipeType.invoke[InscriberRecipe]("inscriber")
}

File: src/main/java/com/algorithmlx/inscribers/init/registry/InscribersRecipeTypes.kt
package com.algorithmlx.inscribers.init.registry

import com.algorithmlx.inscribers.recipe.CastRecipeType
import com.algorithmlx.inscribers.recipe.InscriberRecipe
import net.minecraft.item.crafting.IRecipeType

object InscribersRecipeTypes {
    val inscriberRecipe: IRecipeType<InscriberRecipe> = CastRecipeType.invoke("inscriber").get()
}